### DISCUSSION TITLE
DVClive autoatically adds tracked files to dvc yaml

### DISCUSSION THREAD

**Discussion by JenniferHem** [treeverse/dvc#10399] [General]
Hi all, I have a question regarding DVClive. 

I am running a script via dvc exp run, however this script in itself logs metrics via DVClive. This works perfectly fine except for one thing: It always automatically adds 
```
params:
- modeldata/Experimental/final_models/params.yaml
metrics:
- modeldata/Experimental/final_models/metrics.json
plots:
- modeldata/Experimental/final_models/plots/metrics:
    x: step
```
to my dvc yaml at the end. I would like to add the files only as output to the stage as then the file would be much cleaner, but if I do so it re-adds the lines. Can I prevent this or do I just have to accept that the metrics are added? If so: why is it adding automatically the plots (I am not saving any plots, and I am mostly not using neural networks so steps do not make sense anyway!) could I at least prevent this? 
Using this code to track: 
```
with Live(
    dir=save_path
) as live:
    for col, value in best_performance.items():
        if col == "params":
            for param, param_value in param_dict.items():
                param = param.split("__")[-1]
                live.log_param(param, param_value)
        elif col == "model_name":
            model_type = value.split(" ")[1]
            model_descriptors = value.split(" ")[0]
            live.log_param("model_type", model_type)
            live.log_param("model_descriptor", model_descriptors)
        else:
            live.log_metric(col.replace("test_score", metric_name), value)
    live.log_metric(f"mean_CV_{selection_metric}", abs(CV_score_mean))
    live.log_metric(f"std_CV_{selection_metric}", CV_score_std)
    live.end()
```
Clarification for the tracking loop: I am using something like GridsearchCV and want to track the outcomes later with DVC.

dvclive version: 3.45.0

Thank already for the help!

**Comment by dberenbaum** [comment]
Hi @JenniferHem, you can disable that automatic dvc.yaml tracking using `Live(dvcyaml=False)`.

**Reply by JenniferHem** [reply]
Thanks a lot, I totally misinterpreted this parameter!

**Comment by JenniferHem** [comment]
Sorry a follow up here: the dvc.yaml file stays as expected. but there is still a folder named "plots/metrics" generated which contains steps and the metrics, e.g.:
```
step	mean_CV_neg_mean_squared_error
0	0.30080350966867914
```
can I turn that off?

**Reply by dberenbaum** [reply]
`live.log_metric()` will always add a plot and step value. If you don't want any plots or step tracking, you can instead log metrics via the [summary](https://dvc.org/doc/dvclive/live/make_summary), which writes to `dvclive/metrics.json`, like this:

```python
live.summary[f"mean_CV_{selection_metric}"] = abs(CV_score_mean)
```
